packages/datadog-plugin-http/test/client.spec.js

```diff
@@ -55,7 +55,7 @@ describe('Plugin', () => {
         if (appListener) {
           appListener.close()
         }
-        return agent.close({ ritmReset: false })
+        return agent.close({ ritmReset: false, wipe: true })
```

Comment:
I don't like this change. However when running all the tests together using `PLUGINS="http" yarn test:plugins` we see the following problem:

If we don't wipe everything after the `client.spec.js` has finished running, the following test that runs (`code_origin.spec.js`) will fail because the tracer isn't reloaded with the new tracer config passed into the `agent.load()` call. This seems like a bug in how agent.load works, but I might be wrong.